backend/rent.py

```
@@ -0,0 +1,158 @@
+import mysql.connector
+import bcrypt
+
+app = Flask(__name__)
+app.secret_key = 'MY_SECRET_KEY'
+CORS(app)
+
+def database_connection():
+    return mysql.connector.connect(
+            host = 'localhost',
+            user = 'root',
+            password = 'Spickles@004.com',
```

Ruff (0.12.2)

Possible hardcoded password assigned to argument: "password"

(S106)